scripts/synthetix/facilicate_trade.js

```diff
@@ -0,0 +1,97 @@
+const { getExchange } = require("../utils/trading_strategy_helpers")(web3, artifacts)
+const { SynthetixJs } = require("synthetix-js")
+const { calculateBuyAndSellAmountsFromPrice, fetchTokenInfoFromExchange } = require("../utils/trading_strategy_helpers")(web3, artifacts)
+
+// TODO - put this somewhere more generic. Copied from dex-contracts
+const sendTxAndGetReturnValue = async function (method, ...args) {
```

Comment:
For now it doesn't seem as if you are using the return value, are you?

Reply:
Great point. I wasn't happy about copying this function over from another repo so, may as well eliminate it!